templates/results-row.php

```diff
@@ -14,5 +14,15 @@
 	<td>
 		{{data.message}}
 	</td>
+	<# if ( data.hasLinks ) { #>
+		<td>
+			<# if ( data.link ) { #>
```

Comment:
Do we need both conditions? If `data.link` is true, then certainly `data.hasLinks` must be true also.

Reply:
Thank you, @swissspidy, for your feedback. Regarding the `data.hasLinks` issue, it is always true. However, as an additional check, we've implemented the suggested approach as per the discussion at https://github.com/WordPress/plugin-check/pull/298#discussion_r1373364037.

Reply:
Surely `data.hasLinks` is not always true, otherwise what is the point?

What I am suggesting is

```
<# if ( data.link ) { #>
		<td>
<a href="{{data.link}}" target="_blank">
					<?php esc_html_e( 'View in code editor', 'plugin-check' ); ?>
					<span class="screen-reader-text"><?php esc_html_e( '(opens in a new tab)', 'plugin-check' ); ?></span>
				</a>
		</td>
	<# } #>
```

Because `data.link` is true, `data.hasLinks` has to be true as well.

Reply:
Argh, got it, brain fart 🤦